doc/classes/AnimatedSprite3D.xml

```diff
@@ -83,12 +85,16 @@
 	</methods>
 	<members>
 		<member name="animation" type="String" setter="set_animation" getter="get_animation">
+			The current animation from the [code]frames[/code] resource. If this value is changed, the [code]frame[/code] counter is reset.
```

Comment:
Link to member variables with [member frame], the docs should support this soon.
Passive voice: `If this value is changed` -> `If this value changes`